Lib/dis.py

```diff
@@ -32,20 +32,30 @@ def _try_compile(source, name):
     return c
 
 def dis(x=None, *, file=None, depth=None):
-    """Disassemble classes, methods, functions, generators, or code.
+    """Disassemble classes, methods, functions, and other compiled objects.
 
-    With no argument, disassemble the last traceback.
+       With no argument, disassemble the last traceback.
```

Comment:
Happy to change it, but this convention is inconsistent even within `dis.py`: compare e.g. `_try_compile`, `Instruction` and `_get_name_info`, which align to the right of `"""`, with `_disassemble`, `get_instructions`, and `get_instructions_bytes`, which use the alignment you prefer. I was matching `_try_compile` since it's directly above.

`git blame` sheds no light on this that I can see, since inconsistent alignments were used even within the same patch.

Also, I just confirmed that after applying `trim` from PEP 257 (in 3.x you have to manually add a dummy `maxint` to `sys`), all three docstrings below are equivalent. (PEP 257 only mentions `foo` and `bar`).

~~~~
def foo():
    """A multi-line
    docstring.
    """

def bar():
    """
    A multi-line
    docstring.
    """
    
def baz():
    """A multi-line
       docstring.
    """
~~~~